### PR TITLE
Remove old files

### PR DESCRIPTION
**Pull request by nikolai-b**
Removing old assets and old code.
I don't think we are using any of these images (I searched the codebase).  I also don't think we are using the manual Rmd anymore so I've removed that.

### CODE REVIEW

File: static/manual_body.Rmd [hunk 1]
```diff
@@ -1,237 +0,0 @@
----
```
Comment (usr110) [inline static/manual_body.Rmd]: This script generates the manual_body.html file, so I don't think you can delete it.  (this is where all the recent changes in manual were carried out).

Comment (Robinlovelace) [inline static/manual_body.Rmd]: True that - good to clean up the repo though.

Comment (nikolai-b) [inline static/manual_body.Rmd]: Thanks, I've kept that `Rmd` file.  It looks pretty static so I thought it was just html, apologies.

Comment (usr110) [inline static/manual_body.Rmd]: It's alright. Robin might not like the idea, but I believe we don't know the markdown document any more. HTML will serve the purpose fairly well.

Comment (nikolai-b) [inline static/manual_body.Rmd]: Do you mean don't *need*?

Comment (usr110) [inline static/manual_body.Rmd]: Corrected, thanks - edited the message.

Comment (Robinlovelace) [inline static/manual_body.Rmd]: The reason for using .Rmd, aside from making it easier for others to edit (as stated by @jf1 ), is that we can show how to reproduce the results. Writing this html from scratch would take a long time!

manual-eg

**Pull request merged**